### ISSUE TITLE
Staging Review finding: Confirm pattern for dependent children multi page pattern

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#82850]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Pension benefits
**Product:** 527EZ
**Feature:** Update multi-response (single page list loop) pattern

## Findings details
**VA.gov Experience Standard - issue:** [User encounters design components that are out of date.](https://design.va.gov/about/experience-standards/#user-encounters-up-to-date-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** Yes
**Collab Cycle Reviewer:** @allison0034 (Design), @erinrwhite (IA)

### Description

I noticed the pattern you are using for dependent children is slightly different than what we have in the Design System which is slightly different than what we have in the forms library.

## Recommended action

~~No recommendation right now. I need to get confirmation on what the exact pattern is and then I can update this ticket.~~
Follow the pattern we have in the [Design System.](https://design.va.gov/patterns/ask-users-for/multiple-responses#how-to-design-and-build---multi-page) Add Step 3: One the Veteran selects “Continue” they are moved onto a new screen with the message: “Now we’re going to ask you some follow-up questions about each of your [items]. We’ll go through them one by one.”

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** [Multiple responses](https://design.va.gov/patterns/ask-users-for/multiple-responses)
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by allison0034** [comment]
@matt4su I have updated this ticket. You just need to add the missing step. Let me know if you have any questions.

**Comment by aplatt-coforma** [comment]
Thanks @danbrady for providing the update designs for implementation.
Moving this to Product Backlog column for eng to work next sprint.

**Comment by ToddWebDev** [comment]
Related Issue:
[Multiple Page List & Loop Pattern Summary Page Title UI](https://app.zenhub.com/workspaces/benefits-pension-64e775aaa6b1ca1ed49b2ede/issues/gh/department-of-veterans-affairs/va.gov-team/89286)

**Comment by ToddWebDev** [comment]
This issue can be unblocked and is ready for content and design review.

**Comment by gmaciolek** [comment]
Content and design review complete. Found 2 issues:

1. In the edit flow, the first letter of the dependent child's first name is lowercase in the h3 on every screen, even if it was entered in the field with an uppercase letter. Casing should match what the user entered into the field.
![Image](https://github.com/user-attachments/assets/81c31365-6095-4d7e-aa2c-e8d55b6b2a7a)

2. On the summary page, where we ask "Do you have another dependent child to add?" the "Yes" option is selected by default. Both options should be empty.
![Image](https://github.com/user-attachments/assets/619b20e6-8a72-4ce0-a7c9-192b7c8ea704)

[Figma file for reference](https://www.figma.com/design/9JKK5Eo43uJWEr66JPiebc/Pension-Claim-Form-21P-527EZ?node-id=8092-12469&t=xT5wygcXmRpgM9Os-4)

**Comment by ToddWebDev** [comment]
I've added a ticket to address this issue: 
https://github.com/department-of-veterans-affairs/va.gov-team/issues/93217

**Comment by ToddWebDev** [comment]
This issue can be unblocked and is ready for content and design review.

**Comment by aplatt-coforma** [comment]
Eng work is complete and changes are on staging environment for now. 
Will pick up multiple page list & loop work once it becomes prioritized with HCD.
